content/aips/LUSDETHV3.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Add LUSD to Aave Ethereum V3 pool
+author: Marc Zeller (@marczeller), Aave-Chan initiative
+shortDescription: Onboard LUSD on Aave Eth V3
+discussions: https://governance.aave.com/t/arc-add-lusd-to-ethereum-v3-market/11522
+created: 2023-02-14
+updated: 2023-02-14
+---
+
+## Simple Summary
+This AIP proposal presents Aave with the opportunity to onboard LUSD to the Ethereum v3 Liquidity Pool.
+
+## Motivation
+Supporting stablecoin diversity diversity is part of the Aave-Chan Initiative (ACI) delegate platform.
+
+The ACI does not have any link nor is paid by Liquity or any entity to present this AIP.
+
+The risk parameters provided introduce LUSD as a non-collateral asset.
+
+Supply and borrow caps are suggested for this asset to limit risk exposure, the caps have been proposed by a join collaboration between Gauntlet & Chaos Labs.
+
+## Specification
+
+Ticker: LUSD
+
+Contract Address: [0x5f98805A4E8be255a32880FDeC7F6728C6568bA0](https://etherscan.io/token/0x5f98805A4E8be255a32880FDeC7F6728C6568bA0)
+
+|Risk Parameter|Value|
+| --- | --- |
+|Isolation Mode|NO|
+|Enable Borrow|YES|
+|Enable Collateral|NO|
+|Loan To Value|N/A|
+|Liquidation Threshold|N/A|
+|Liquidation Bonus|N/A|
+|Reserve Factor|10%|
+|Liquidation Protocol Fee|N/A|
```

Comment:
there's a mismatch with https://github.com/aave/aip/pull/286/files#diff-783520fb0bbb52e6d3b4e4fa9140f78b41da6c003cb6e9741fefc20b00af4199R66 - i think setting to 10% is fine though.

Reply:
Fixed, thanks.